### ISSUE TITLE
Problem accessing higher res HDRIs

### ISSUE THREAD

**Issue by photojourneyman** [Poly-Haven/polyhavenassets#10]
I'm having an issue where I update an HDRI to a higher resolution, it seems to update OK but then I get a python error and it only uses the 1K HDRI in the render.  I have checked my file paths and it should be OK. Any ideas on what to try? I have attached some screenshots.
![polyhavenError](https://user-images.githubusercontent.com/88125078/188808024-a2b0a555-677c-4de3-b89c-3ee2185adc9b.jpg)

**Comment by gregzaal** [comment]
I'm not sure why this would happen - does it do this for all HDRIs or just a particular one?

If it happens to all of them, then I'll need your help to diagnose the issue. I won't be able to fix it without being able to reproduce it myself, so I need an exact list of steps to take to reproduce the error from scratch, starting from factory defaults.

**Comment by xcodz-dot** [comment]
I did some reading of the code and have traced back the issue to line 37 and 38. Most probabbly, there is an image attribute but it is set to None which I suspect that is not expected behaviour. I would suggest to change

```python
        if hasattr(node, "image"):
            if node.image is not None:
                yield node.image
```

It is a temprorary fix. I could not go any deeper without more information.

**Comment by photojourneyman** [comment]
Thanks for that and sorry for not getting back to you earlier.  I managed to find a fix by re-setting to a new poly haven folder location and reloading the initial files. Something must have got a bit mixed up when I did my initial install on my PC desktop. It works fine on my Macbook pro. All is working well now.

**Comment by gregzaal** [comment]
This is likely caused by unused image nodes (with no image selected) being accidentally added to the tree. Image nodes with generated images (e.g. UV grid) that don't have a filepath would also be an issue. Thanks for the pointer @xcodz-dot - fixed in ac40ca216
